_data/toc.yaml

```diff
@@ -1100,10 +1100,10 @@ manuals:
     title: Getting started
   - path: /compose/swarm/
     title: Using Compose with Swarm
-  - path: /compose/env-file/
-    title: Environment file
   - path: /compose/environment-variables/
     title: Environment variables in Compose
+  - path: /compose/env-file/
+    title: Environment variables in .env file
```

Comment:
@charleswhchan The re-ordering makes sense. Thank you. Could you retain the original title `Environment file` for this entry?

Reply:
Fixed. Please check commit b36210d again. Thanks.